### ISSUE TITLE
Give comments and suggestions to the deck draft

### ISSUE THREAD

**Issue by lam3lka** [VIKING-GARAGE/operations#30]
**Is your feature request related to a problem? Please describe.**
I already have some feedback on the deck, but I would love to know your feedback, especially on vision (1st slide) and numbers (projections). 

**Describe the solution you'd like**
We need to create a deck that shows everyone that VIKING GARAGE will be a success no matter what (that market is big enough and we target the right customer). This presentation should show that if someone will invest return from the investment is guaranteed and we already have the plan how to use funds in next rounds (even if we have no idea if we want to run next rounds of raising funds ;p) Also, we need to clearly show why VIKING GARAGE is a solution (why booking with us is a future) – insurances, reviews, selection of bikes – in a clear way. 

I was also thinking about the visual side – generally, people like it because it is different, still need to work on visibility and coherence. 

How can we improve? 

- [x] Slide 1: a mission statement – how we want to define our 'why'

- [x] Slide 7: a problem statement (I didn't get any feedback on that, but maybe you have some ideas how it should look like)

- [x] Slide 14: a solution (slide 4 is our one-liner), but we need to work on clear explanation why we are different (Slide 5), why we are better, why there is no other way to book bike, why we are future

- [x] Slide 18: show sharing economy market size versus peer-to-peer car/ machines rentals – I have a feeling that we need to be more specific in terms of market

If we will do 12M revenue by 2020, our market share of moto rentals in the motorcycle industry would be 11% – I changed slide and in my opinion, it should look like this:

<img width="767" alt="zrzut ekranu 2018-09-17 o 15 02 03" src="https://user-images.githubusercontent.com/28931514/45609459-ab7ab900-ba8a-11e8-9adc-25757ad79fd2.png">

- [x] I want to show 5 chosen tourist destinations and show how many tourists are eager to rent a bike there, based on that make a projections 

- [x] We need to show customer acquisition cost versus lifetime customer value (this one will be easy) and the best performing channels

- [ ] Slide 19: Forecast, please comment here. 

Data from: https://docs.google.com/spreadsheets/d/1z_oot2bqGc4blC3U4uXq20xaMGH_rMIZrOA5g_dwqus/edit#gid=0 

Forecasts from the graphs: 
1st year - 2019: around $250000 revenue, 2 175 rentals + $200000 gear rentals & experiences
2nd year - 2020: around $12M revenue, 1M rentals + $2M gear rentals & experiences
3rd year - 2021: around $50 M revenue + $5M gear rentals & experiences

- [x] Slide 22: I am not sure how to show our competition, right now it shows how we are different than others but still, we could improve it.

- [x] Slide 25: I will add how we want to use funds (pie chart)

- [ ] I will add a slide about the exit (how much was invested in Sharing economies so far and what was the average return from investment) – good case practice from one of my mentors but for a short deck, not the presentation on Demo Day.

Link to the deck: https://drive.google.com/file/d/1L3gCjNs0NYjkft53jYuegmV5aa-7PeM7/view

**Describe alternatives you've considered**

**Additional context**

**Comment by michalmikolajczyk** [comment]
@lam3lka I am sorry can you provide the link? The description does not seem to fit https://drive.google.com/open?id=1L3gCjNs0NYjkft53jYuegmV5aa-7PeM7

**Comment by lam3lka** [comment]
@michalmikolajczyk updated

**Comment by michalmikolajczyk** [comment]
Slide 1 – how about:

Ride the motorcycle you want

- [x] Resolved

**Comment by michalmikolajczyk** [comment]
Slide 4:

A motorcycle is the best way
to explore wilderness of the world.

Slide 5:

However, **existing options** to rent motorcycles, 
are **limited, dangerous and expensive**

- [x] Resolved

**Comment by michalmikolajczyk** [comment]
- [x] slides 12, 14

HOW IT WORKS

For travelers

We give travelers the freedom to rent any motorcycle they want, wherever they want.

• rent **reviewed** motorcycles from locals with (public profiles) (nie jestem pewny jak napisać)
• usually at least **30% cheaper** compared to typical rental companies
• get access to a community of riders using the platform, locals and travelers alike

**Comment by michalmikolajczyk** [comment]
- [x] slides 13, 14

For bike owners

We empower motorbike owners to make their bikes sustainable. (trochę słaba gramatyka chyba)

• Put up offers for motorcycle rentals and tours
• Increase sales by reaching online customers
• Fully utilize the resources #ridethemdonthidethem
• Increase quality, gain reputation, get more customers and support

**Comment by michalmikolajczyk** [comment]
- [x] slide 18

Globally, the sharing economy has
been growing faster than Facebook, Google, and Yahoo combined.

In 2018 PwC projected, that the revenue of all sharing economy markets combined,
will be $335 billion, equal to that of traditional retails markets.

---

Asia please check if the above is actually correct and provide footnote reference (link)

**Comment by michalmikolajczyk** [comment]
- [x] Regarding sales:
If we make $12 millions in 2020, then our share in the motorcycle industry will be around 0.1% because the total industry is about $116 billions

**Comment by michalmikolajczyk** [comment]
- [x] destinations

I showed the pitch deck to a friend of mine, and he told me that we did not mark the locations correctly. He told me e.g. that in Borneo, there's nothing but jungle, but we should have added dots to North Borneo. Let's double check all locations.

**Comment by michalmikolajczyk** [comment]
- [x] customer acquisition cost vs lifetime value

Is there anyway we could do subscriptions? perhaps not charge subscribers our fee or something?

**Comment by michalmikolajczyk** [comment]
- [ ] prediction slides – discuss, correct grammar around 12 USD

**Comment by michalmikolajczyk** [comment]
Slide 22 is really nice Asia!

**Comment by michalmikolajczyk** [comment]
- [x] Slide 9

VIKING GARAGE is a marketplace
with motorcycle rental offers,
by a community of trusted bike owners,
for travelers worldwide.

**Comment by michalmikolajczyk** [comment]
- [x] Slide 12

please add spaces to surround all the highlighted words, so the highlights have margins

**Comment by michalmikolajczyk** [comment]
- [ ] slide 12

where do you have the _30%_ and _$300_ from?

**Comment by michalmikolajczyk** [comment]
- [x] slide 19

typo:
Chiang Mai (now it's Chaing Mai)

**Comment by lam3lka** [comment]
$300 = $10 (average bike price per day)* 30 days
30% I took prices from typical rental companies like Bali Bike Rental to compare how much travelers can save

**Comment by lam3lka** [comment]
> * [x]  slide 12
> 
> where do you have the _30%_ and _$300_ from?

$300 = $10 (average bike price per day)* 30 days
30% I took prices from typical rental companies like Bali Bike Rental to compare how much travelers can save

**Comment by lam3lka** [comment]
> Slide 1 – how about:
> 
> Ride the motorcycle you want
> 
> * [ ]  Resolved

I changed it. Now we have: "ride a motorcycle you want, wherever you want"

**Comment by michalmikolajczyk** [comment]
nope, it should be "the"

**Comment by michalmikolajczyk** [comment]
The articles referenced below, from Entrepreneur.com, helped me prepare the Executive Summary. You already apply a lot of the techniques, but perhaps they can be useful.

https://www.entrepreneur.com/article/247312
https://www.entrepreneur.com/article/247256
https://www.entrepreneur.com/article/246776

**Comment by michalmikolajczyk** [comment]
- [x] add 1 or 2 slides showing screenshots from the app – how it looks like, the list of offer, single offer

**Comment by michalmikolajczyk** [comment]
- [x] improve "Team". We should have a more professional appearance I think.

**Comment by lam3lka** [comment]
> * [ ]  customer acquisition cost vs lifetime value
> 
> Is there anyway we could do subscriptions? perhaps not charge subscribers our fee or something?

Yes, or give access to a premium services e.x. content or events. 

But here I mean subscription fee paid by bike owners.

**Comment by lam3lka** [comment]
> * [ ]  prediction slides – discuss, correct grammar around 12 USD

Ohh sorry, didn't get it. What do you mean?

**Comment by michalmikolajczyk** [comment]
- [x] Team slide

Michal Mikolajczyk

https://bs-uploads.toptal.io/blackfish-uploads/leaders_page/content/leaders_section/leader/avatar_file/avatar/3181/Michal_Mikolajczyky-e7f5a67df067e22a0e795d2190c2143f.jpeg

Founder and CEO
Software Engineer
Community Leader @ Toptal

**Comment by lam3lka** [comment]
> * [x]  Team slide
> 
> Michal Mikolajczyk
> 
> https://bs-uploads.toptal.io/blackfish-uploads/leaders_page/content/leaders_section/leader/avatar_file/avatar/3181/Michal_Mikolajczyky-e7f5a67df067e22a0e795d2190c2143f.jpeg
> 
> Founder and CEO
> Software Engineer
> Community Leader @ Toptal

<img width="763" alt="zrzut ekranu 2018-09-25 o 23 23 12" src="https://user-images.githubusercontent.com/28931514/46024749-0743e280-c11a-11e8-9ae5-842c27715b5b.png">

**Comment by lam3lka** [comment]
> * [x]  add 1 or 2 slides showing screenshots from the app – how it looks like, the list of offer, single offer

<img width="753" alt="zrzut ekranu 2018-09-25 o 23 25 24" src="https://user-images.githubusercontent.com/28931514/46024914-52f68c00-c11a-11e8-93d9-8bef194aab14.png">

**Comment by michalmikolajczyk** [comment]
- [ ] add slide 27 with motorcycle sales per region

**Comment by lam3lka** [comment]
[VG Pitch Deck Keynote [2].pdf](https://github.com/VIKING-GARAGE/operations/files/2447079/VG.Pitch.Deck.Keynote.2.pdf)
@michalmikolajczyk added
